py/dml/traits.py

```diff
@@ -348,8 +348,11 @@ def mktrait(site, tname, ancestors, methods, params, sessions,
     # referencing 'dev.xyz' from a shared method is always OK, even if
     # it's technically an untyped object parameter
     reserved_symbols.pop('dev', None)
-    return Trait(site, tname, ancestors, methods, params, sessions,
-                 ancestor_vtables, ancestor_method_impls, reserved_symbols)
+    trait = Trait(site, tname, ancestors, methods, params, sessions,
+                  ancestor_vtables, ancestor_method_impls, reserved_symbols)
+    if tname == 'object':
+        dml.globals.object_trait = trait
```

Comment:
side-effect, please move upward, probably to mkglobals

Reply:
That _might_ be too late, but I can try.

Reply:
No issue!